pkg/rtc/participant.go

```diff
@@ -1673,6 +1682,8 @@ func (p *ParticipantImpl) addMigrateMutedTrack(cid string, ti *livekit.TrackInfo
 }
 
 func (p *ParticipantImpl) addMediaTrack(signalCid string, sdpCid string, ti *livekit.TrackInfo) *MediaTrack {
+	p.requireBroadcast = true
```

Comment:
I think this needs to be locked by `p.lock` ?

Reply:
yep, good catch